src/Components/PrivateRout/PrivateRoute.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Redirect, Route} from "react-router-dom";
+
+function PrivateRoute({auth, children, ...rest}) {
+    return (
+        <Route {...rest} >
+            {auth === true ? children : <Redirect to="/login"/>}
```

Comment:
Je stuurt hier al een true of false mee. Dit kun je dus versimpelen naar {auth ? children : <Redirect to="/login"/>}